common/lib/xmodule/xmodule/modulestore/tests/test_semantics.py

```diff
@@ -0,0 +1,119 @@
+from xmodule.modulestore.tests.utils import PureModulestoreTestCase, VersioningModulestoreBuilder, MongoModulestoreBuilder, SPLIT_MODULESTORE_SETUP
+from xmodule.modulestore.exceptions import ItemNotFoundError
+from xmodule.modulestore import ModuleStoreEnum
+from xmodule.modulestore.tests.factories import CourseFactory
+
+class AboutItemSemantics(PureModulestoreTestCase):
+    """
+    Verify the behavior of About items (which are intentionally orphaned, DIRECT_ONLY/autopublished)
+    blocks intended to store snippets of course content.
+    """
+
+    __test__ = False
+
+    def setUp(self):
+        super(AboutItemSemantics, self).setUp()
+        self.course = CourseFactory.create(
+            org='test_org',
+            number='999',
+            run='test_run',
+            display_name='My Test Course',
+            modulestore=self.store
+        )
+
+        self.about_usage_key = self.course.id.make_usage_key('about', 'video')
+
+    def assertAboutDoesntExist(self):
+        with self.assertRaises(ItemNotFoundError):
+            self.store.get_item(self.about_usage_key, revision=ModuleStoreEnum.RevisionOption.published_only)
+        with self.assertRaises(ItemNotFoundError):
+            self.store.get_item(self.about_usage_key, revision=ModuleStoreEnum.RevisionOption.draft_only)
+
+        with self.store.branch_setting(ModuleStoreEnum.Branch.draft_preferred):
```

Comment:
Because, unfortunately, they're handled by different code paths. I wanted to be sure that either way you specify the branch, you get the right thing.

Ideally, we'd go down to only having one way of specifying the branch...